packages/web-components/@sfx/search-box/stories/index.ts

```diff
@@ -1,68 +1,78 @@
+  const showClearButton = boolean('Show clear button', true) ? 'clearbutton' : '';
+  return (
+    '<sfx-search-box\n' +
+    (placeholder ? ` placeholder="${placeholder}"\n` : '') +
+    (showSearchButton ? ` ${showSearchButton}\n` : '') +
+    (showClearButton ? ` ${showClearButton}\n` : '') +
+    '></sfx-search-box>'
+  );
+}
 
 storiesOf('Components|Searchbox', module)
   .addDecorator(withKnobs)
-    .add(
-      'Default - with search and clear button',
-      () => `
-    <sfx-search-box searchbutton clearbutton></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    )
-    .add(
-      'Without any SFX provided buttons',
-      () => `
-    <sfx-search-box></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    )
-    .add(
-      'With a clear button, without a search button',
-      () => `
-    <sfx-search-box clearbutton></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    )
-    .add(
-      'With custom placeholder text',
-      () => `
-    <sfx-search-box placeholder="${text(
-      'Placeholder Title',
-      'Placeholder here...'
-    )}"></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    );
+  .add(
+    'Rendering based on events and attributes',
+    () => {
+      const searchBoxComponent = getSearchBoxComponent();
+      return `
+      ${searchBoxComponent}
+      ${getDisplayCode(searchBoxComponent)}
+      `;
+    },
+    {
+      customEvents: updateTextEvent,
+      notes: {
+        markdown: `
+        ${searchboxNotesMarkdownIntro}
+
+          * The SF-X Search Box component rendering with customizations.
+            * Clear and search buttons are optional.
+              * If the 'clearbutton' attribute is present, a clear button will display.
+              * If the 'searchbutton' attribute is present, a search button will display.
+                * To add and remove these attributes:
+                  1. Visit the **Knobs** tab and toggle the 'Show search button' and 'Show clear button'.
+                    * View the DOM and component update accordingly.
+            * The placeholder text can be customized.
+                * If the 'placeholder' attribute is present with a string value, the placeholder text will replace the default placeholder (Type your search)
+                  * To modify the 'placeholder' attribute:
+                    1. Visit the **Knobs** tab and modify the text contained within the 'Placeholder Text' field.
+                      * View the component and DOM update accordingly.
+          * The SF-X Search Box component populating with an updated search term in response to the \`sfx::update_search_term\` event.
```

Comment:
Also use the event constant.

```suggestion
          * The SF-X Search Box component populates with an updated search term in response to the \`sfx::update_search_term\` event.
```